src/node.cc

```diff
@@ -3720,6 +4079,9 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--trace-events-enabled") == 0) {
       trace_enabled = true;
     } else if (strcmp(arg, "--trace-event-categories") == 0) {
+      // XXX(sam) change trace_enabled_categories to std:string so it manages
```

Comment:
I think its too unrelated to this PR, I'll PR it seperately. I'm tempted to pull the preload_modules refactor  infot a std::vector out into its own PR, too, its not really related to this, even though I needed it to allow `-r` in NODE_OPTIONS